docs/docs/target/kubernetes.md

```diff
@@ -101,13 +91,79 @@ Total: 3 (UNKNOWN: 0, LOW: 1, MEDIUM: 0, HIGH: 2, CRITICAL: 0)
 └────────────────┴────────────────┴──────────┴────────┴───────────────────┴──────────────────────────────────┴───────────────────────────────────────────────────┘
 ```
 
+### Taints and Tolerations
+
+The node-collector scan-job will run on every node. In case the node has been tainted, it is possible to add toleration to the scan job for it to be scheduled on the tainted node
+
+- `--tolerations  key1=value1:NoExecute,key2=value2:NoSchedule` this flag wil enable node-collector to be schedule on tainted Node
+
+Example:
+
+```sh
+trivy k8s --report summary --tolerations  key1=value1:NoExecute,key2=value2:NoSchedule
+```
+
+### Non-intrusive
+
+You can control whether the node scan-job (`node-collector`) will run in the cluster. To disable it, add the `--non-intrusive` flag  
+
+- `--non-intrusive` This flag will exclude findings related to Node (infra assessment) misconfigurations
+
+By default, the node scan-job (`node-collector`) will run in the cluster.
+
+Example:
+
+```sh
+trivy k8s --report summary --non-intrusive
+```
+
+### Skip-images
+
+You can control whether Trivy will scan and download the cluster resource images. To disable this feature, add the --skip-images flag.
```

Comment:
People might be worried that Trivy will download and keep all of the images from the cluster on their local machine. Should we elaborate on the process? I mean, immediate questions would be something like: if someone has permission in the cluster to run the scan, will they have access to all the images? Should they have access to run the scan but not the images -- I mean someone would need access to all namespaces etc. to perform a scan of all images? Maybe we should add to each section what kind of permission the user requires in the first place to run the scan successfully, e.g. if Bob does not have access to Alice's namespace, will he still be able to scan her container images?

Reply:
@AnaisUrlichs  for public images I assume its not relevant and if it is private registry images then the user will have to provide credential